hack/deploy_fast.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+check_ocp() {
+    local KUBECONFIG_PATH=$1
+
+    export KUBECONFIG=$KUBECONFIG_PATH
+    
+    if oc get nodes; then
```

Comment:
Fine with merging this initially, but there is a (pretty good) chance we try to deploy this test on a broken cluster, which will quickly add up with false fast-test failures.

Maybe a good strategy is:
- Add improved assessments of cluster health (i.e. nodes are ready, all cluster operators are ready and working)
- If I try to run a fast-test on a broken cluster, it should run as a slow test (redeploy the whole thing).

False failures might increase the burden on the contributor to track down whether their code broken something, or whether the CI is broken.

Reply:
totally agree.... I have to figure out ways I can add cluster-health-checks to this.

Reply:
Or we could dedicate a server to fast-tests which will:
- redeploy lastest OCP nightly on a schedule
- if failing cluster state is detected, re-install OCP

Reply:
@SalDaniele I discussed the idea of having a server dedicated to fast tests with @bn222. The issue with that is that we want our CI running (having 1 server can create a potential issue where fast tests are done, but slow tests are in queue - we want to avoid that scenario by pooling them together)